CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # PRQL Changelog
 
+## 0.5.0 — [unreleased]
```

Comment:
This can't be left for 0.5.0 and has to be included in 0.4.2 because currently the playground doesn't match otherwise.

Reply:
Ah. But it's a breaking change, it cannot go into 0.4.2.

I think we need to freeze the playground & docs to the latest release.

Reply:
Totally agree with you on both points.

How would 0.4.2 work though because it's already merged onto main so wouldn't it be included in 0.4.2 anyway?

How much work do you think it would be to maintain release branches and have those rebased automatically with each merged PR?

Reply:
I think this is viable. 

I also think we can just do 0.5.0 next if we have a breaking change, rather than 0.4.2?